app/common/helpers/collections.py

```diff
@@ -435,6 +435,30 @@ def get_previous_question(self, current_question_id: UUID) -> Optional["Question
 
         raise ValueError(f"Could not find a question with id={current_question_id} in collection={self.collection}")
 
+    def get_summary_line_for_add_another(self, component: "Component", *, add_another_index: int) -> str:
+        if not component.add_another_container:
+            raise ValueError("summary lines can only be generated for components in an add another container")
+
+        questions = []
+        if component.add_another_container.is_group:
+            if component.add_another_container.presentation_options.add_another_summary_line_question_ids:
+                for qid in component.add_another_container.presentation_options.add_another_summary_line_question_ids:
+                    try:
+                        questions.append(self.get_question(qid))
+                    except ValueError:
+                        pass
+            questions = questions or cast("Group", component.add_another_container).cached_questions
+        else:
+            questions = [cast("Question", component)]
+
+        answers = []
+        for question in questions:
+            answer = self.cached_get_answer_for_question(question.id, add_another_index=add_another_index)
+            if answer:
+                answers.append(answer.get_value_for_text_export())
```

Comment:
🤏  tiny wonder if this should be rendering the answer template instead, so that it shows up consistent with how these answers display otherwise in eg check your answers or submission pages?

Not totally sure it's compatible - but would be worth exploring that, I think, if you haven't already?